spark-commons/src/test/scala/za/co/absa/spark/commons/errorhandling/implementations/ErrorMessageArrayTest.scala

```diff
@@ -197,6 +192,25 @@ class ErrorMessageArrayTest extends AnyFunSuite with SparkTestBase {
     val result = resultDfToResult(resultDf)
 
     assert(result == expected)
+  }
+
+  test("errorColumnAggregationType should return an ArrayType structure for column added during the aggregation") {
+    val errColName = "errCol"
```

Comment:
Just to test, the override works 😉 
```suggestion
    val errColName = "specialErrCol"
```